test/controllers/measurements_controller_test.rb

```diff
@@ -41,6 +41,86 @@ class MeasurementsControllerTest < ActionDispatch::IntegrationTest
     assert_response :created
   end
 
+  test 'should create measurement without created_at and use current timestamp' do
+    travel_to Time.zone.parse('2023-01-01 12:00:00') do
+      assert_difference('Measurement.count') do
+        post measurements_url, params: { measurement: { sensor_id: @measurement.sensor.id,
+                                                        temperature: 20.5 } },
+                               env: private_auth_header
+      end
+
+      new_measurement = Measurement.last
+      assert_in_delta Time.current, new_measurement.created_at, 1.second
```

Comment:
If you use `travel_to` time should be frozen, no? Then `created_at` should be exactly `2023-01-01 12:00:00`

Reply:
Yeah, I think you're right, that would be easier to read.